agents/src/llm/llm.ts

```diff
@@ -48,16 +59,56 @@ export abstract class LLM {
 }
 
 export abstract class LLMStream implements AsyncIterableIterator<ChatChunk> {
+  protected output = new AsyncIterableQueue<ChatChunk>();
   protected queue = new AsyncIterableQueue<ChatChunk>();
```

Comment:
The two queues are kind of confusing. If I'm a user implementing a plugin, I would not be sure which one to use